### ISSUE TITLE
FastMember - Default Sort Order

### ISSUE THREAD

**Issue by XtremeOwnageDotCom** [rebus-org/Tababular#5]
By default, the fast member library will sort column names by NAME, instead of order specified.

https://github.com/mgravell/fast-member/blob/master/FastMember/MemberSet.cs

As a result, you will need to specify your column order, by ensuring column names are alphabetically ordered.

**Comment by XtremeOwnageDotCom** [comment]
Recommend adding some method to change the default sorting order.

Here is the exact snippet of code, producing undesired results:

```csharp
members = type.GetProperties(PublicInstance).Cast<MemberInfo>().Concat(type.GetFields(PublicInstance).Cast<MemberInfo>()).OrderBy(x => x.Name)
                .Select(member => new Member(member)).ToArray();
```

**Comment by tgckpg** [comment]
I second this.

**Comment by frankhommers** [comment]
+1
I third this ;-)

**Comment by mookid8000** [comment]
fixed in Tababular 3.0.1, which is on nuget.org in a few minutes
